src/lib/DomainData.ml

```diff
@@ -65,12 +65,14 @@ and tp =
   | Sg of tp * Ident.t * tp_clo
   | Nat
   | TpAbort
+  | TpHCom of dim * dim * cof * con
 
 and hd =
   | Global of Symbol.t
   | Var of int (* De Bruijn level *)
   | Coe of con * dim * dim * con
   | HCom of cut * dim * dim * cof * con
+  (* C*HM: add Cap of dim * dim * cof * con * cut *)
```

Comment:
If anyone is bored and wants to write some code for fun, try adding this constructor and following the rest of the `C*HM` trail 😎